homeassistant/components/influxdb/const.py

```diff
@@ -75,7 +75,7 @@
 BATCH_BUFFER_SIZE = 100
 LANGUAGE_INFLUXQL = "influxQL"
 LANGUAGE_FLUX = "flux"
-TEST_QUERY_V1 = "SHOW SERIES LIMIT 1;"
+TEST_QUERY_V1 = "SHOW DATABASES;"
```

Comment:
Did we have a test query for the sensor platform before the refactor?

Reply:
Yes. `SHOW SERIES LIMIT 1;` was the one that was there. The reason it worked before is because each sensor has its own `InfluxDBClient` instance and so each did its own test on creation of the sensor. Since each sensor has a database this was all fine.

But one of the things I did in the refactor was change it so the same `InfluxDBClient` instance is shared among all sensors. This meant I had to do the test before the sensors are setup. Which means I don't necessarily know what bucket/database to connect to, the user may have configured that per sensor (like what happened in the linked issue). So I need a test that can be run just against Influx itself and not against any particular database.

Reply:
Note that I also think in the end this new query will be an improvement. I am already working on adding back in the logic that only sets up sensors pointing to a bucket/database the user can see. Now with this query I can do that with only the output of a single query whereas before it was doing one query per sensor for that check.

Reply:
Sound good!

This test query has changed two times over the years. First change was due to too heavy query and second change was due to admin permissions required. How does `SHOW DATABASES` measure in that regard?

Reply:
Yea I was worried about that and I actually think this is the ideal one in this spot. I was looking up how Influx handles permissions and it says specifically that non-admin users can do a `SHOW DATABASES` to see the databases they have READ and/or WRITE permissions on. So not only does that make it a good connectivity test in this spot, I can actually use the output to validate the databases listed on queries.

Here's the doc for reference: https://docs.influxdata.com/influxdb/v1.8/administration/authentication_and_authorization/#non-admin-users

Reply:
Good!

Reply:
Oh and your second question, its very very light.  It literally just returns a dictionary with one field per database the user can see. It looks like this against my testing V1 instance:
```
>>> list(influx.query("SHOW DATABASES;").get_points())
[{'name': '_internal'}, {'name': 'Test'}, {'name': 'AnotherDB'}]
```